lib/App/DuckPAN/Web.pm

```diff
@@ -63,8 +63,15 @@ sub BUILD {
 			}
 			$share_dir_hash{$_->module_share_dir} = ref $_ if $_->can('module_share_dir');
 			$path_hash{$_->path} = ref $_ if $_->can('path');
+
+			my $alt_rewrites = $_->alt_rewrites;
```

Comment:
I think this should be inside the `if ($_->does('DDG::IsSpice')) {` block? Only Spice will have the `alt_to`